SafeguardSessions/README.md

```diff
@@ -59,61 +60,61 @@ connection was closed: Could not establish trust relationship for the SSL/TLS
-Reproduce the error with trace logging enabled and create a technical case described in the [Troubleshooting](#troubleshooting) section.
+A response returned by SPS does not contain a field that the Power BI Connector requires for processing data.
+Reproduce the error with trace logging enabled, and create a technical case, as described in the [Troubleshooting](#troubleshooting) section.
 Attach the mashup trace logs to the issue.
 
 **B) The source IP interpreted a malformed request.**
 
-Your filter might be invalid. Make sure you input your filter parameters correctly. For more information on the available search fields, see **List of available search queries** in the **One Identity Safeguard for Privileged Sessions Administration Guide** on the [Technical documents for One Identity Safeguard for Priviledged Sessions] page.
+Your filter might be invalid. Make sure you specify your filter parameters correctly. For more information on the available search fields, see **List of available search queries** in the **One Identity Safeguard for Privileged Sessions Administration Guide** on the [Technical documents for One Identity Safeguard for Priviledged Sessions] page.
 
 **C) The username or password you have specified is invalid.**
 
 Make sure you use a local user to access SPS with a valid username and password. One Identity recommends creating a dedicated local user for the purpose of importing data from SPS into Power BI.
 
 **D) You are not authorized to access the specified resource.**
 
-In order to access audit data, the user you use to access SPS needs to have search access rights.
+In order to access audit data, the user you use to access SPS must have search access rights.
 
 **E) The requested resource is not found.**
 
-In order to fetch audit data from SPS, the Connector relies on advanced search that uses database snapshots to ensure data consistency throughout a data fetch. If for some reason the database snapshot disappears, this error can occur. Try to initiate the data fetch process again.
+In order to fetch audit data from SPS, the Power BI Connector relies on the advanced search method, which uses database snapshots to ensure data consistency throughout data fetching. If for some reason, the database snapshot disappears, this error can occur. To resolve the issue, try initiating the data fetching again.
 
 **F) Snapshot quota exceeded.**
 
-This happens if multiple users want to use advanced search relying on database snapshots at the same time. In this case, try initiating the data fetch at least 5 minutes later so that an existing snapshot may expire in the system and a new one can be opened.
+This happens if multiple users want to use advanced search relying on database snapshots at the same time. In this case, try initiating the data fetching process at least 5 minutes later so that an existing snapshot expires in the system, and a new one can be opened.
```

Comment:
With the introduction of snapshot caching, this part is not neccessarily true. Is the "user" aware that he is identified by the IP and the remote agent and not the username?